src/plugin/modules/widgets/genomes/kbaseGenomeWideOverview.js

```diff
@@ -33,7 +33,7 @@ define([
             this.render();
             return this;
         },
-        render: function () {
+        render: function (genomeData) {
```

Comment:
Does anything call this with `genomeData`? The `init` function above just calls it without arguments. Or is this a placeholder for later?